StoreTestWPF.Presentation/ResourceDictionary.xaml

```diff
@@ -0,0 +1,22 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+                    xmlns:local="clr-namespace:StoreTestWPF.Presentation"    
+                    xmlns:userControls="clr-namespace:StoreTestWPF.Presentation.Controls"
+                    xmlns:viewModels="clr-namespace:StoreTestWPF.ViewModel.ViewModels;assembly=StoreTestWPF.ViewModel"  
+                    xmlns:utils="clr-namespace:StoreTestWPF.ViewModel.Utils;assembly=StoreTestWPF.ViewModel"  
+                    xmlns:materialDesign="http://materialdesigninxaml.net/winfx/xaml/themes" >
+    <utils:PriceConverter x:Key="PriceConverter" />
+    <BooleanToVisibilityConverter x:Key="BoolToVis" />
+    <DataTemplate DataType="{x:Type viewModels:CakeViewModel}">
+        <materialDesign:Card>
+            <userControls:CakePreview
```

Comment:
Do you remember our agreement about DataContext?

Reply:
Fixed